spec/helpers/schema_helper_spec.rb

```diff
@@ -274,6 +274,11 @@
         data = valid_data.merge(dietary_requirements: "Foo")
         expect(validate_against_form_response_schema(data).first).to include("dietary_requirements")
       end
+
+      it "does not return a list of errors when carry_supplies has a nil value" do
```

Comment:
Typo in test name. Should be `dietary_requirements`.

Reply:
Also, shouldn't the test on line 268: "returns a list of errors when dietary_requirements is missing" fail now?

Reply:
We're not allowing the value to be missing, only to allow it to have a nil value.  Even if the user doesn't answer the later food questions, the value will always be nil: https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/358/files#diff-8c917c651898f2137b6f3b59a44e43f2R32-R33.